mlflow/deployments/databricks/__init__.py

```diff
@@ -0,0 +1,44 @@
+from mlflow.deployments import BaseDeploymentClient
+
+
+class DatabricksDeploymentClient(BaseDeploymentClient):
+    def create_deployment(self, name, model_uri, flavor=None, config=None, endpoint=None):
+        raise NotImplementedError
+
+    def update_deployment(self, name, model_uri=None, flavor=None, config=None, endpoint=None):
+        raise NotImplementedError
+
+    def delete_deployment(self, name, config=None, endpoint=None):
+        raise NotImplementedError
+
+    def list_deployments(self, endpoint=None):
+        raise NotImplementedError
+
+    def get_deployment(self, name, endpoint=None):
+        raise NotImplementedError
+
+    def predict(self, deployment_name=None, inputs=None, endpoint=None):
+        raise NotImplementedError
+
+    def create_endpoint(self, name, config=None):
+        raise NotImplementedError("TODO")
+
+    def update_endpoint(self, endpoint, config=None):
+        raise NotImplementedError("TODO")
+
+    def delete_endpoint(self, endpoint):
+        raise NotImplementedError("TODO")
+
+    def list_endpoints(self):
+        raise NotImplementedError("TODO")
+
+    def get_endpoint(self, endpoint):
+        raise NotImplementedError("TODO")
+
+
+def run_local(name, model_uri, flavor=None, config=None):
```

Comment:
Build a local serving container and validate the capacity to return inference predictions? Is that what this is? (if so, this is awesome)

Reply:
I'm actually not sure what this is.

Reply:
The target_help implementation as explained in the ABC is definitely out of scope for a Databricks plugin (not entirely sure what that would return even if there was an available endpoint to target?). The run_local might also be a "maybe nice to have in the far-off future", but definitely something that would be rather challenging to simulate model serving behavior from within OSS.